opentelemetry-sdk/src/trace/span_processor.rs

```diff
@@ -316,6 +316,7 @@ impl BatchSpanProcessor {
         let handle = thread::Builder::new()
             .name("OpenTelemetry.Traces.BatchProcessor".to_string())
             .spawn(move || {
+                let _suppress_guard = Context::enter_telemetry_suppressed_scope();
```

Comment:
Suppression for simple processor will be added as separate PR?

Reply:
Can't do it until https://github.com/open-telemetry/opentelemetry-rust/issues/2871 is resolved. "touching" context in On_End panics today!

Reply:
Thanks for context on the context issue :) add TODO with link to issue?

Reply:
https://github.com/open-telemetry/opentelemetry-rust/issues/2883